packages/eslint-config-eslint/README.md

```diff
@@ -32,7 +32,7 @@ In your `.eslintrc` file, add:
 
 ### Where to ask for help?
 
-Join our [Mailing List](https://groups.google.com/group/eslint) or [Chatroom](https://eslint.org/chat)
+Open a [discussion](https://github.com/eslint/eslint/discussions) or stop by our [Discord server](https://eslint.org/chat) instead of filing an issue.
```

Comment:
In this file, there's another one here:

https://github.com/eslint/eslint/blob/9b2fcf7e928fc92ac6d43617bdee1bda250b7491/packages/eslint-config-eslint/README.md?plain=1#L5

Reply:
In `links.json` file, there is one occurrence.
https://github.com/eslint/eslint/blob/337f7ed96131d873be7ae6b010739476d0ad15e9/docs/src/_data/links.json#L5